### DISCUSSION TITLE
[question] using YaNco in Powershell

### DISCUSSION THREAD

**Discussion by olfolfolf** [dbosoft/YaNco#238] [Q&A]
Could you give a small example, how to connect with YaNco in Powershell?

**Comment by fw2568** [comment]
Hello,

short answer: don't think that it would make fun using YaNco in Powershell. Technically it will work somehow, as Powershell is .net under the hood, but we use a lot of function lambda' s. 
But if you really want to try, I would be happy if you share you results.

Best Regards,
Frank
